build.rs

```diff
@@ -95,6 +95,15 @@ fn main() {
         "periph_gpio",
         "periph_i2c",
         "periph_spi",
+        "periph_uart",
+        "periph_uart_collision",
+        "periph_uart_hw_fc",
+        "periph_uart_modecfg",
+        "periph_uart_nonblocking",
+        "periph_uart periph_uart_modecfg",
```

Comment:
Was this intentional?

Reply:
No, that was my lack of proper self-review :-(

Reply:
:smirk: you know, you can ask other to review... :smirk: